pmd-core/pom.xml

```diff
@@ -190,6 +195,12 @@
             <artifactId>system-rules</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
+            <artifactId>jackson-annotations</artifactId>
+            <version>2.6.0</version>
+            <scope>compile</scope>
+        </dependency>
```

Comment:
```suggestion
```

I think this is unused now

Reply:
Thanks @oowekyala for looking into this. We will remove the dependency very quickly. Did you check the rest of the PR? And like it?

Reply:
Yes this is nice, it's well tested and documented :)

Reply:
Nice catch! Removed.